docker-compose.yml

```diff
@@ -2,21 +2,21 @@ version: "3.9"
 
 services:
   postgres:
-    image: postgres
+    image: postgres:15
     volumes:
       - postgres_data:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: libretime
-      POSTGRES_PASSWORD: libretime # Change me !
+      POSTGRES_USER: ${POSTGRES_USER:-libretime}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-libretime} # Change me !
     healthcheck:
       test: pg_isready -U libretime
 
   rabbitmq:
-    image: rabbitmq:alpine
+    image: rabbitmq:3.11-alpine
     environment:
-      RABBITMQ_DEFAULT_VHOST: /libretime
-      RABBITMQ_DEFAULT_USER: libretime
-      RABBITMQ_DEFAULT_PASS: libretime # Change me !
+      RABBITMQ_DEFAULT_VHOST: ${RABBITMQ_DEFAULT_VHOST:-/libretime}
```

Comment:
Maybe document these in the docker setup docs?

Reply:
Done, hope this is enough. The docker-setup is a bit more complex than the install script, I am unsure if we want to make it easier by scripting it. Maybe some bash command in the docs is enough.

Reply:
Once legacy is removed or support env vars instead of the configuration file, we will be able to make this simpler and remove the duplication between the env file and the config file.